### PR TITLE
fix(docker): fix goaws docker , add syncserver docker

### PR DESCRIPTION
**Pull request by deeptibaghel**

### CODE REVIEW

File: _scripts/goaws.sh [hunk 1]
```diff
@@ -1,23 +1,3 @@
...
-
-else
-  is_up=$(docker ps -a | grep pafortin/goaws | grep Up | cut -d' ' -f1)
-  if [ -z "$is_up" ]; then
-    docker start $container_id
-    echo "starting $container_id"
-    while true; do sleep 1000; done
-  fi
-fi
+docker run --rm --name goaws -p 4100:4100 pafortin/goaws
```
[the remark this answers is not in the document]
Comment (deeptibaghel) [reply _scripts/goaws.sh]: I do this :
docker ps -a 
docker stop CONTAINER_ID
docker rm CONTAINER_ID

File: _scripts/install_all.sh [hunk 2]
```diff
@@ -48,7 +48,7 @@ cd fxa-basket-proxy; npm i; cd ..
 
 cd 123done; npm i; CONFIG_123DONE=./config-local.json node ./scripts/gen_keys.js; cd ..
 
-cd syncserver; make build; cd ..
+cd syncserver; docker build -t syncserver:latest .; cd ..
```
[the remark this answers is not in the document]
Comment (deeptibaghel) [reply _scripts/install_all.sh]: I thought so but then i never searched the hub , will do it.

**Comment by deeptibaghel** [comment]
@vladikoff syncserver seems to be failing in travis, should i test it in linux ?

**Comment by vladikoff** [comment]
@deeptibaghel sure, i wonder what it is ...

**Comment by deeptibaghel** [comment]
@vladikoff travis failure was because it needed prefix "mozilla" here :
docker run --rm \
  -p 5000:5000 \
  -e SYNCSERVER_PUBLIC_URL=http://localhost:5000 \
  -e SYNCSERVER_SECRET=5up3rS3kr1t \
  -e SYNCSERVER_SQLURI=sqlite:////tmp/syncserver.db \
  -e SYNCSERVER_BATCH_UPLOAD_ENABLED=true \
  -e SYNCSERVER_FORCE_WSGI_ENVIRON=false \
  **mozilla**/syncserver:latest \
  /usr/local/bin/gunicorn --bind 0.0.0.0:5000 \
  syncserver.wsgi_app

**Comment by vladikoff** [comment]
Works well @deeptibaghel !! 👍

**Pull request merged**